subprojects/junit-jupiter/junit-jupiter.gradle

```diff
@@ -0,0 +1,19 @@
+description = "Mockito JUnit 5 support"
+
+apply from: "$rootDir/gradle/java-library.gradle"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    compile project.rootProject
+    testCompile libraries.assertj
+    testCompile libraries.junitJupiterLauncher
+    implementation libraries.junitJupiterEngine
+}
+
+test {
+    useJUnitPlatform()
```

Comment:
Gradle team call Junit Jupiter "JUnit platform". I'm curious why? Should we follow this naming convention?

Reply:
**JUnit Platform** supports many engines, like "JUnit Jupiter (JUnit 5)", or "JUnit Vintage (JUnit 3 and 4)" or other `TestEngine` implementations, like those listed here: https://github.com/junit-team/junit5/wiki/Third-party-Extensions#junit-platform-test-engines

**JUnit Jupiter** is an implementation of the `TestEngine` interface ... which provides extension support. Here is where the Mockito Extension is living.

More details at: https://junit.org/junit5/docs/current/user-guide/#overview-what-is-junit-5